platform/ui/src/components/HotkeysPreferences/HotkeysPreferences.tsx

```diff
@@ -30,7 +30,7 @@ const HotkeysPreferences = ({
   const splitedHotkeys = splitHotkeyDefinitionsAndCreateTuples(visibleHotkeys);
 
   if (!Object.keys(hotkeyDefinitions).length) {
-    return 'No hotkey definitions';
+    return t('No hotkey definitions');
```

Comment:
Should we add this to the UserPreferencesModal.json files that you are comfortable with?

Reply:
There is already existing "No hotkeys found" key in "User Preferences Modal". I'll fix it in the next commits